packages/lesswrong/components/community/modules/CommunityMembers.tsx

```diff
@@ -133,7 +133,14 @@ const styles = createStyles((theme: ThemeType): JssStyles => ({
   },
   description: {
     color: theme.palette.grey[800],
+    display: '-webkit-box',
+    "-webkit-line-clamp": 2,
+    "-webkit-box-orient": 'vertical',
+    overflow: 'hidden',
     marginTop: 12,
+    [theme.breakpoints.down('xs')]: {
+      "-webkit-line-clamp": 4,
+    }
```

Comment:
Since we're not using htmlBio anymore, I can use line clamp to limit the height of the bio text